.azure/linux-stack.yml

```diff
@@ -69,7 +69,6 @@ jobs:
       source .azure/linux.bashrc
       stack build --stack-yaml $(YAML_FILE) --test --bench --only-dependencies
       stack install --stack-yaml $(YAML_FILE) # `hie` binary required for tests
-      stack --stack-yaml $(YAML_FILE) exec hoogle generate
```

Comment:
Seems like a mistake to delete this?

Reply:
No, it actually crashed Azure CI and we dont use it yet for the tests. 
Maybe someone else wants to tackle this problem, but it does not seem to be urgent.